### ISSUE TITLE
moduleForComponent stopped working in ember-source@3.13.0-beta.1

### ISSUE THREAD

**Issue by ro0gr** [emberjs/ember.js#18305]
A simple `moduleForComponent`-style render test, like:

```js
import { moduleForComponent, test } from 'ember-qunit';
import hbs from 'htmlbars-inline-precompile';

moduleForComponent('test', 'Integration | test', {
  integration: true
});

test('Test', async function(assert) {
  this.render(hbs`test`);

  assert.ok(true);
});
```

leads to the following error:
> TypeError: Cannot read property 'lookup' of undefined

which is raised from https://github.com/emberjs/ember.js/blob/66ed9abd529d3673bec176c5279d42b8ecd87dd1/packages/%40ember/-internals/glimmer/lib/template.ts#L39

This error occurs starting from ember-source@3.13.0-beta.1 and still persists in ember-source@3.13.0-beta.3.

Here is a repo with reproduction - https://github.com/ro0gr/ember-moduleForComponent-owner

**Comment by rwjblue** [comment]
Thank you for reporting! I believe https://github.com/emberjs/ember-test-helpers/pull/677 would fix, but I wanted to actually dig into things first to make sure we don’t make unneeded changes to the “legacy” testing system (if we can instead mitigate in Ember itself).

**Comment by ro0gr** [comment]
I've checked, the issue is fixed with emberjs/ember-test-helpers#677. Thanks!

**Comment by rwjblue** [comment]
Thank you for confirming!
